packager/VisibilityChecker.cc

```diff
@@ -30,6 +30,58 @@ static core::SymbolRef getEnumClassForEnumValue(const core::GlobalState &gs, cor
     return core::Symbols::noSymbol();
 }
 
+static void reportMissingTestImport(const core::Context &ctx, const core::packages::PackageInfo &package,
```

Comment:
This name was slightly misleading to me before I read the implementation of the function:

It's not saying that the `test_import` is missing, it's saying that the `test_import` is there but it needs to be an `import` to be able to use the constant in non-test code.